### DISCUSSION TITLE
Logging options

### DISCUSSION THREAD

**Discussion by stevedrees** [devpi/devpi#888] [General]
I am looking for documentation about the logging options for devpi-server. I want to send the logs to a file when devpi-server is run under supervisor and it isn't obvious to me how to accomplish that.

**Comment by stevedrees** [comment]
Answering my own question.

I added a stdout_logfile directive to the supervisord config file for devpi-server. That solves my immediate problem, but if there are better options for this I am open to them.

**Comment by fschulze** [comment]
I would always recommend to use the logging facilities of your process manager, as you already found out yourself.

There is ``--logger-cfg`` which parses a json or yaml file and passes it on to [``dictConfig``](https://docs.python.org/3/library/logging.config.html?highlight=dictconfig#logging.config.dictConfig) if you want to fine tune the logging.
